### PR TITLE
Split up sources by templates

### PR DESCRIPTION
**Pull request by rmarianski**
This allows specifing start and stop zooms within a particular source, on a template level. The python code will take the templates for a particular source, and merge them together with a `UNION ALL`.

Some of the tests are currently failing. Will still need to look into those.

### CODE REVIEW

File: queries.yaml [hunk 1]
```diff
@@ -11,21 +11,126 @@ all:
...
+    - template: ne-water-playas-10m.jinja2
+      zoom_start: 5
+      zoom_stop: 9
+
   osmdata:
-    template: osmdata.jinja2
-    start_zoom: 8
+
+    - template: osmdata-earth.jinja2
+      zoom_start: 9
```
Comment (nvkelso) [inline queries.yaml]: 8?

File: queries.yaml [hunk 2]
```diff
@@ -11,21 +11,126 @@ all:
...
+
+    - template: osmdata-water.jinja2
+      zoom_start: 9
```
Comment (nvkelso) [inline queries.yaml]: 8?

File: queries/ne-places.jinja2 [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+SELECT
+
+  gid AS __id__,
+
+  {% filter geometry %}the_geom{% endfilter %} AS __geometry__,
+  NULL::bytea AS __label__,
+
+  {{ ne_common_properties() }},
+
+  NULL::jsonb AS __boundaries_properties__,
```
Comment (zerebubuth) [inline queries/ne-places.jinja2]: Just checking - we still need these `NULL::jsonb AS __layer_properties__` to make the columns match across the `UNION ALL`s?

File: queries.yaml (hunk 1 above)
Comment (rmarianski) [inline queries.yaml]: Nice catch! That explains why the tests were failing :) When I rebased off master I didn't catch that this changed upstream.

Fixed in b8ed8e6.

File: queries.yaml (hunk 2 above)
Comment (rmarianski) [inline queries.yaml]: This too. Also updated in b8ed8e6.

File: queries/ne-places.jinja2 (hunk 3 above)
Comment (rmarianski) [inline queries/ne-places.jinja2]: Yes, I just double checked and it errors out if these are missing.

**Comment by rmarianski** [comment]
Because I missed some of the changes in #1359, do you mind double checking to see if there's anything else in there that I could have overlooked? The tests that @zerebubuth presciently added now pass, but just wanted to verify in case there was some logic in there that might not have been captured by a test.

**Pull request merged**